apps/betterangels-admin/src/app/clients/apollo/csrf.ts

```diff
@@ -0,0 +1,87 @@
+import { ApolloLink, Observable } from '@apollo/client';
+import { eraseCookie, getCookie } from '../../utils/storage/cookies';
+
+const csrfCookieName =
```

Comment:
We can make the constants optionally overridable to the fetchClient and appolloClient after we move to a shared library, rather than relying on environment overrides in a constants file.